modules/dist/pom.xml

```diff
@@ -158,62 +117,6 @@
         <jboss.home>${project.build.directory}/wildfly-${jboss.version}</jboss.home>
       </properties>
     </profile>
-    <!--
-    Name:  deploy
-    Descr: Deploy stack on WildFly instance
-    -->
-    <profile>
-      <id>deploy</id>
-      <activation>
-        <property>
-          <name>!nodeploy</name>
-        </property>
-      </activation>
-      <build>
-        <plugins>
-          <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-assembly-plugin</artifactId>
-              <executions>
-                  <execution>
-                      <id>distribution-package</id>
-                      <phase>prepare-package</phase>
-                      <goals>
-                          <goal>single</goal>
-                      </goals>
-                      <configuration>
-                          <finalName>assembly</finalName>
-                          <ignoreDirFormatExtensions>true</ignoreDirFormatExtensions>
-                          <appendAssemblyId>false</appendAssemblyId>
-                          <descriptors>
-                              <descriptor>src/main/scripts/assembly-deploy-artifacts.xml</descriptor>
```

Comment:
so this PR should contain removal of this file too?

Reply:
Yes. This is replaced with the jbossws-galleon-feature pack.